@types/annotator/index.d.ts

```diff
@@ -44,16 +35,16 @@ declare module 'annotator' {
 
   export namespace ui {
     export interface IAnnotationLoader extends widget.Widget {
-      load(ann: IAnnotation, position: util.IPosition): JQuery.Deferred<IAnnotation>;
+      load(ann: any, position: util.IPosition): JQuery.Deferred<any>;
     }
 
     export namespace highlighter {
       export class Highlighter extends widget.Widget {
         constructor(element: Element);
-        public drawAll(anns: IAnnotation[]): void;
-        public draw(ann: IAnnotation): void;
-        public undraw(ann: IAnnotation): void;
-        public redraw(ann: IAnnotation): void;
+        public drawAll(anns: any[]): void;
+        public draw(ann: any): void;
+        public undraw(ann: any): void;
+        public redraw(ann: any): void;
```

Comment:
ogółem `any` jest niemile widziane w TypeScripcie (tzw. bad practice 😛 ), czemu `IAnnotation` było nie ok?

Reply:
Właściwie jedynym faktycznym (a nie logicznym) powodem było to, że chciałem, żeby w AnnotatoinViewModel `ranges` było słownikiem, a ponieważ w modelu w bazie jest stringowym JSonem (i w API), IAnnotationAPIModel `ranges` jest stringiem.